contrib/dokku-installer.py

```diff
@@ -100,7 +101,10 @@ def do_POST(self):
         with open('{0}/HOSTNAME'.format(dokku_root), 'w') as f:
             f.write(params['hostname'].value)
 
-        command = ['sshcommand', 'acl-add', 'dokku', 'admin']
+        user = 'admin'
+        if self.admin_user_exists():
+            user = 'web-admin'
+        command = ['sshcommand', 'acl-add', 'dokku', user]
         for key in params['keys'].value.split("\n"):
             proc = subprocess.Popen(command, stdin=subprocess.PIPE)
             proc.stdin.write(key)
```

Comment:
Right but the web ui allows splitting the keys via newline, and as new sshcommand doesn't allow duplicate usernames, we'll need to modify the user for each successive write.